### ISSUE TITLE
No logging output

### ISSUE THREAD

**Issue by odalet** [vcsjones/AzureSignTool#121]
I'm using version 2.0.17 of the tool and, with an invalid set of arguments on the command line, I end up with... no error displayed at all. I can only know something went wrong by examining `ERRORLEVEL`.

I managed to get console logging working by modifying a bit the end of `Program.Main`: I changed

```csharp
return application.Execute(args);
```

into:

```csharp
var rc =  application.Execute(args);
serviceProvider.Dispose();  
return rc;
```

Disposing the service provider has the effect of flushing the logger and then my wrong arguments give me:

```
fail: AzureSignTool.Program[0]
      Options -kvi, -kvs are required.
```

I'd be glad to turn this into a PR, however, I've noticed the code changed quite a lot since [v2.0.17 tag](https://github.com/vcsjones/AzureSignTool/tree/v2.0.17). So, I'm not sure if I should branch from the tag, and into what branch this would be reintegrated...

**Comment by vcsjones** [comment]
I think this has been addressed in 3.0.0. can you please give it a try?

**Comment by odalet** [comment]
Indeed, it's fixed. Thanks!
